src/services/procosysApi.ts

```diff
@@ -73,20 +73,20 @@ const procosysApiService = ({ axios, apiVersion }: ProcosysApiServiceProps) => {
         return data as string[];
     };
 
-    const searchForCommPackage = async (
+    const searchForMcPackage = async (
```

Comment:
Kan vi bruke samme funksjon for alle fire søkene? Altså bare kalle den "search" eller "getSearchResults", tar inn en "searchType"-prop og så bytter den "McPkg"-biten av stringen ut med searchtypen vi passer inn?